AsyncDisplayKit/ASTableNode.m

```diff
@@ -79,11 +79,20 @@ - (void)didLoad
 
 - (void)updateCurrentRangeWithMode:(ASLayoutRangeMode)rangeMode
 {
-    if (!self.isNodeLoaded) {
-        return;
-    }
-    
-    [self.view.rangeController updateCurrentRangeWithMode:rangeMode];
+  if (!self.isNodeLoaded) {
+    return;
+  }
+  
+  [self.view.rangeController updateCurrentRangeWithMode:rangeMode];
+}
+
+- (void)clearCurrentRangeModeUpdate
```

Comment:
Can we just use updateCurrentRangeWithMode: for this?  Because there are not many modes, I think it is fairly unlikely that an app would really need to be able to return to the prior default value without being able to set one.  Main goal is to reduce API surface area, as although this is still in +Beta, we should converge it towards something that we open up more broadly.